src/main/python/rlbot/parsing/rlbot_config_parser.py

```diff
@@ -70,12 +70,13 @@ def parse_configurations(config_parser: ConfigObject, config_location, config_bu
         match_config.player_configs.append(player_config)
 
     extension_path = config_parser.get(RLBOT_CONFIGURATION_HEADER, EXTENSION_PATH_KEY)
-    if extension_path:
+    if extension_path and extension_path != 'None':  # The string 'None' ends up in people's config a lot.
```

Comment:
Could we also fix the source of this problem please?

Reply:
Yes I'd like to fix that too. Gonna save it for a future PR though since we're gonna need this little band-aid regardless.